source/npm/publish.js

```diff
@@ -4,7 +4,7 @@ const {from} = require('rxjs');
 const {catchError} = require('rxjs/operators');
 const handleNpmError = require('./handle-npm-error');
 
-const pkgPublish = (pkgManager, options) => {
+const getPkgPublishArgs = options => {
```

Comment:
I'd rename this to `getPackagePublishArgs`

Reply:
Even better: `getPackagePublishArguments`